CHANGELOG.md

```diff
@@ -5,63 +5,85 @@ CHANGELOG
 ------
 
 **ENHANCEMENTS**
-- Add possibility to use an existing Instance Profile for cluster creation and Imagebuilder.
-- Support restart/reboot for instance type with instance store (ephemeral drives).  
-- Add possibility to use an existing Private Route53 Hosted Zone when using Slurm as scheduler.
-- Add possibility to suppress validators and filter validation failures by failure level at cluster creation and update time.
-- Add new set of commands to build and manage Custom AMIs.
-- Add new set of commands to list, view and export cluster and image builder logs.
-- Split head node and compute fleet instance roles and add possibility to configure a different instance role
-  for each queue.
+- Add support for pcluster actions (e.g., create-cluster, update-cluster, delete-cluster) through HTTP endpoints
+  with Amazon API Gateway.
+- Add support to use an existing Instance Profile for cluster creation and custom AMI creation.
+- Support restart/reboot for EC2 instance types with 
```

Comment:
Enable the possibility to restart / reboot the head node also for instance types with instance store(...). Those operations remain anyway managed by the user that is responsible for the status of the cluster while operating on the head node.